modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -204,11 +206,20 @@ export default {
         ? this.imageRelationshipComponent
         : 'AposRelationshipEditor';
 
+      const imgInfos = item.attachment ? {
+        url: item.attachment._urls.original,
+        width: item.attachment.width,
```

Comment:
These would be the width and height of the entire attachment, not the existing crop. I think you are trying to get at the existing crop here.